resources/lang/en/events/death/general.json

```
@@ -3062,10 +3064,11 @@
             "war"
         ],
         "tags": [
+            "no_body",
             "all_lives"
         ],
         "weight": 20,
-        "event_text": "m_c was sealed in a cramped rabbit burrow by o_c_n warriors. {PRONOUN/m_c/subject/CAP} didn't make it out alive.",
+        "event_text": "m_c was sealed in a cramped rabbit burrow by o_c_n warriors and couldn't get out.",
```

damb. dark. Not sure it's clear that cat is defo dead tho.
```suggestion
        "event_text": "m_c was buried alive in a cramped rabbit burrow by o_c_n warriors.",
```